great-idea/css/index.css

```diff
@@ -63,4 +63,218 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+* {
+	box-sizing: border-box;
+}
+
+html, body {
+	/* font-family: 'Bai Jamjuree'; */
+	font-size: 17px;
```

Comment:
That's an interesting choice for a front size.

Reply:
Haha visually it was appropriate.